packages/expo-contacts/android/src/main/java/expo/modules/contacts/Contact.java

```diff
@@ -183,6 +186,21 @@ public String getLastName() {
         return lastName;
     }
 
+    @Nullable
+    public String getDisplayName() {
+      if (displayName == null) {
+        if (firstName != null) {
+          if (lastName != null) {
```

Comment:
Too many `if`s 😅 Maybe we can make it simpler? Also, what if `firstName == null`? Shouldn't we return `lastName` then?

Reply:
I made it simpler ;) 

We shouldn't return only `lastName`, cause then Android will interpret this value as a first name.